File: godfather/factions/neutral/survivor.py
from godfather.game import Player
from godfather.factions.base import Faction


class SurvivorNeutral(Faction):
    name = 'Survivor'
    id = 'neutral.survivor'
    win_con = 'Live until the end of the game.'

    @property
    def category_name(self):
        return 'Neutral'

    def has_won_independent(self, player: Player):
        return player.is_alive

File: src/lib/mafia/factions/neutral/Survivor.ts
import Faction from '@mafia/structures/Faction';
import type Player from '@mafia/structures/Player';

export default class SurvivorFaction extends Faction {
	public name = 'Survivor';
	public winCondition = 'game/factions:survivorWinCondition';
	public independent = true;

	public hasWonIndependent(player: Player) {
		return player.isAlive;
	}
}
